tests/assets/smb_example_config.yml

```diff
@@ -0,0 +1,26 @@
+root-dir: ~/Documents/HSR/semester_06
```

Comment:
So - I know we discussed a lot of this in [EPJ-39](https://jira.keltec.ch/jira/browse/EPJ-39), but now that I've read the code and tried to write a config by hand (without looking at the example too much), I noticed a lot of "bad" definitions here.

Sorry for the nitpicking here, @ThunderKey @bytinbit - but I think getting good names here is a very important usability-thing for kitovu (probably *the* most important before we have a sophisticated configuration GUI) - so I added comments below.

Reply:
You're right. What about this:
```yaml
root-dir: ~/Documents/HSR/semester_06

global-ignore:
  - Thumbs.db
  - .DS_Store

connections:
  - name: skripte-server
    plugin: smb
    # hostname: svm-c213.hsr.ch
    # port: 445
    # share: skripte
    # domain: HSR
    username: example_user
  # ...

subjects:
  - name: Engineering-Projekt
    connections:
      - connection: skripte-server
        ignored:
          - SubDir
          - example.txt
        remote-dir: Informatik/Fachbereich/Engineering-Projekt/EPJ
        # local-dir: Engineering-Projekt
```

@The-Compiler @bytinbit any thoughts?